insights/parsers/tests/test_freeradius.py

```diff
@@ -0,0 +1,152 @@
+import pytest
+
+from insights.parsers import SkipException
+from insights.parsers.freeradius import FreeRadius
+from insights.tests import context_wrap
+
+RADIUSD_CONF_FILE = "etc/raddb/radiusd.conf"
+RADIUSD_CONF_CONTENT = """
+# -*- text -*-
+##
+## radiusd.conf -- FreeRADIUS server configuration file - 3.0.13
+
+prefix = /usr
+exec_prefix = /usr
+sysconfdir = /etc
+localstatedir = /var
+sbindir = /usr/sbin
+
+log {
+        #  The command-line option "-X" over-rides this option, and forces
+        #  logging to go to stdout.
+        #
+        destination = files
+}
+""".strip()
```

Comment:
There's no point in stripping this string if it's going to be handled as line content that is stripped anyway.